code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -142,9 +142,9 @@ func (sc *StorageSmartContract) updateBlobber(
 		return fmt.Errorf("error calculating staked capacity: %v", err)
 	}
 
-	if blobber.Capacity < stakedCapacity {
-		return fmt.Errorf("write_price_change: staked capacity(%d) exceeding total_capacity(%d)",
-			stakedCapacity, blobber.Capacity)
+	if blobber.SavedData < stakedCapacity {
```

Comment:
Used can be less than staked capacity

Reply:
Oh yes, I was checking if used capacity is not getting less than new staked capacity. Let me make the changes.